app/lib/features/events/widgets/events_calendar.dart

```diff
@@ -78,7 +39,7 @@ class EventsCalendar extends ConsumerWidget {
                       style: Theme.of(context).textTheme.bodyLarge,
                     ),
                     subtitle: Text(
-                      formatDt(e),
+                      formatDt(e) + '\t UTC (GMT+0)',
```

Comment:
shouldn't we format in the local timezone?

Reply:
Should we ? That's possible but designs preview UTC date. I guess I could modify this utility.